### DISCUSSION TITLE
Options for other Javascript runtimes

### DISCUSSION THREAD

**Discussion by sigaloid** [TriliumNext/Trilium#3245] [General]
For example, could bun be used as an alternative runtime to be quicker? 

Edit: Status of bun is blocked because of no async_hooks: https://github.com/oven-sh/bun/issues/798#issuecomment-1346168745

**Comment by zadam** [comment]
Bun claims some compatibility with node.js, so maybe (but personally have doubts).

One major source of trouble would be probably `better-sqlite3` which requires node ABI.

**Reply by sigaloid** [reply]
Bun has a SQLite package designed for it, maybe that could be used instead: https://bun-docs.vercel.app/en/bun_sqlite
